src/Page/SWHeroes.purs

```diff
@@ -0,0 +1,67 @@
+module Page.SWHeroes where
+
+import Prelude
+import Capability.Navigate (class Navigate)
+import Capability.SWHeroes (class SWHeroes, getHeroes)
+import Capability.SWHeroes as SWHeroes
+import Component.HTML.Utils as HU
+import Component.Link (link)
+import Data.Foldable (intercalate)
+import Data.Maybe (Maybe(..))
+import Data.Newtype (unwrap)
+import Data.Route (Route(..))
+import Halogen as H
+import Halogen.HTML as HH
+import Tailwind as TW
+
+data Action
+  = Initialize
+
+data State
+  = Init
+  | Loaded (Array SWHeroes.Response)
```

Comment:
Might be worth using: https://pursuit.purescript.org/packages/purescript-remotedata/4.2.0 ?